src/pages/Category.vue

```diff
@@ -60,8 +60,14 @@ export default {
       products: {},
       category: {},
       filters: { // filters should be set by category, and should be synchronized with magento
-        color: ['red', 'blue'],
-        size: ['XS', 'S', 'M', 'L', 'XL'],
+        color: [{'id': 165, 'label': 'red'}, {'id': 166, 'label': 'blue'}],
+        size: [
```

Comment:
I've put the key-value pairs here as these are just test data. Or should they be placed in resources/models folder while leaving these fields empty to be filled later?

Reply:
Seems just fine for me. I'll add some db-logic to load the appropriate attribute options in here. It's fine for me